MobiusCore/Source/MobiusController.swift

```diff
@@ -211,9 +206,11 @@ public final class MobiusController<Model, Event, Effect> {
                 var disposables: [Disposable] = [loop]
 
                 if let viewConnectable = stoppedState.viewConnectable {
-                    let viewConnection = viewConnectable.connect { [unowned loop] event in
-                        // Note: loop.unguardedDispatchEvent will call our flipEventsToLoopQueue, which implements the
-                        //       assertion “unguarded” refers to, and also (of course) flips to the loop queue.
+                    let viewConnection = viewConnectable.connect { [weak loop] event in
+                        guard let loop = loop else {
+                            MobiusHooks.errorHandler("\(Self.debugTag): cannot use invalid consumer", #file, #line)
```

Comment:
This scenario is already prevented by the `AsyncDispatchQueueConnectable`, but I've added it here just in case the `viewConnectable` type ever changes and doesn't account for it.

Reply:
Another explanation that might be helpful to live alongside the code.